Xlink hd24 2024updates (#685)
haddocking/haddocking.github.io: education/HADDOCK24/XL-MS-oligomer/index.md

[DisVis: Quantifying and visualizing accessible interaction space of distance-restrained biomolecular complexes](https://doi.org/doi:10.1093/bioinformatics/btv333){:target="_blank"}.
  _Bioinformatics_ *31*, 3222-3224 (2015).

> * R.V. Honorato, M.E. Trellet, B. Jiménez-García1, J.J. Schaarschmidt, M. Giulini, V. Reys,  P.I. Koukos, J.P.G.L.M. Rodrigues, E. Karaca, G.C.P. van Zundert, J. Roel-Touris, C.W. van Noort, Z. Jandová, A.S.J. Melquiond and A.M.J.J. Bonvin.

Reviewer: ```suggestion
* R.V. Honorato, M.E. Trellet, B. Jiménez-García, J.J. Schaarschmidt, M. Giulini, V. Reys,  P.I. Koukos, J.P.G.L.M. Rodrigues, E. Karaca, G.C.P. van Zundert, J. Roel-Touris, C.W. van Noort, Z. Jandová, A.S.J. Melquiond and A.M.J.J. Bonvin.
```

indicating that your run has been successfully submitted.
While performing the search, the DisVis web server will update you on the progress of the
job by reloading the status page every 30 seconds.
The runtime of this example case is usually below 5 minutes on our local CPU and grid GPU servers. However the load of the server as well as
pre- and post-processing steps might substantially increase the time until the results are available.

The default on the server is to perform a `quick scanning` grid search (meaning `15.00°` rotational sampling and `2.0Å` grid) in order to get results in a reasonable time.
You can however choose to perform a `complete scanning`, which should give more reliable results (`9.72°` rotational sampling and `1.0Å` grid).

<hr>

## Analysing the DisVis results

**Web server output**

Once your job has completed, and provided you did not close the status page, you will be automatically redirected to the results
page (you will also receive an email notification).

<details style="background-color:#DAE4E7">
<summary>See solution</summary>
If you don't want to wait for your run to complete, you can access the precalculated results of a run submitted

> with the same input and complete scanning [here](https://wenmr.science.uu.nl/disvis/run/rth_3XwWbRcd){:target="_blank"}.

Author: ```suggestion
with the same input and complete scanning <a href="https://wenmr.science.uu.nl/disvis/run/rth_3XwWbRcd" target="_blank">here</a>.
<br>
You can also <a href="https://www.bonvinlab.org/education/HADDOCK24/XL-MS-oligomer/DisVis_output_complete.zip" download>download the corresponding archive here</a>.
```

</details>

## Bonus II: Predicting the oligomeric state with AlphaFold3

The newest version of AlphaFold (AlphaFold3) has recently been published: [Abramson J, __et al._ __Accurate structure prediction of biomolecular interactions with AlphaFold 3.__ _Nature. 2024_](https://ww.nature.com/articles/s41586-024-07487-w).
This new version is not only an improved one compared to its predecessor in terms of accuracy and speed, but it also allows us to now deal with supplementary biochemical entities such as nucleotides, post-translational modifications and some lipides, glycans and co-factors.
While the source code is not yet available for local installations, a dedicated webserver is provided to use the tool.

> It currently allows generating up to 20 modeling procedures per days, which is sufficient for the purpose of this tutorial.

Reviewer: ```suggestion
It currently allows to generate up to 20 predictions per day, which is sufficient for the purpose of this tutorial.
```

> To run it, go to [https://golgi.sandbox.google.com/](https://golgi.sandbox.google.com/).

Reviewer: ```suggestion
To run Alphafold3, go to [https://golgi.sandbox.google.com/](https://golgi.sandbox.google.com/).
```

Pre-calculated AlphFold3 predictions are provided here.
The single zip archive contains the four different runs tested.
Each run consits of an archive containing information about the run and five predicted models (the naming indicates the rank).

* [`AlphaFold3Server-runs.zip`](/education/HADDOCK24/XL-MS-oligomer/AlphaFold3Server-runs.zip)
  - `fold_dimer.zip`: contains the dimer run results (copies = 2)
  - `fold_trimer.zip`: contains the trimer run results (copies = 3)
  - `fold_tetramer.zip`: contains the tetramer run results (copies = 4)
  - `fold_pentamer.zip`: contains the pentamer run results (copies = 5)

### Analysis of the generated models

Once the run is finished and the status reached succeeded, you can click on one of the table rows to go to the provided visualizing tool.

On the top, best model metrics will be shown (ipTM and PTM), and in the visualizer, residues are colored based on their pLDDT values.

> Take time to look at the generated models and the arrangement of the various monomers. When submitting our prediction, we only defined the number monomers, but not the symmetry.

Reviewer: ```suggestion
Take time to look at the generated models and the arrangement of the various monomers. When submitting our prediction, we only defined the number of monomers, but not the symmetry.
```
